src/index.md

```diff
@@ -6,7 +6,7 @@ last_reviewed_at: ""
 This Playbook is our reference for who we are and the way we do things.
 Something canonical that tells us what the current “right way” to do things is.
 
-If you’re a current or potential client, this Playbook is also for you. To help
+If you’re a current or potential client, this Playboosk is also for you. To help
```

Comment:
Tsk tsk

Reply:
I'm impressed by the speed of your review, but I'm afraid this was just me testing the mechanism. I wasn't clear on whether the pull request was created immediately after you click save (it is) or whether you have to click the publish button.

Reply:
Yeah, I don't find the relationship between the actions in the Netlify/Decap CMS and what happens in GitHub that intuitive. I think they make more sense when you don't require PR approval and just use their in-built flow